client/post-editor/post-editor.jsx

```diff
@@ -242,31 +240,9 @@ const PostEditor = React.createClass( {
 		this.hideDrafts();
 	},
 
-	renderNotice: function() {
-		var arrowLink;
-
-		if ( ! this.state.notice || ! this.state.notice.text ) {
-			return;
-		}
-
-		if ( this.state.notice.link ) {
-			arrowLink = (
-				<NoticeAction href={ this.state.notice.link } external={ true }>
-					{ this.state.notice.action }
-				</NoticeAction>
-			);
-		}
-
-		return (
-			<Notice
-				status={ 'is-' + this.state.notice.type }
-				showDismiss={ this.state.notice.type === 'success' ? false : true }
-				onDismissClick={ this.onNoticeClick }
-				className="post-editor__notice"
-				text={ this.state.notice.text }>
-				{ arrowLink }
-			</Notice>
-		);
+	hideNotice: function( event ) {
+		event && event.preventDefault();
```

Comment:
Good catch :)

This code was like this before, but I guess the "boy-scout rule" applies here.